tests/model_registry/model_catalog/conftest.py

```diff
@@ -59,3 +62,34 @@ def updated_catalog_config_map(
 @pytest.fixture(scope="class")
 def expected_catalog_values(request: pytest.FixtureRequest) -> dict[str, str]:
     return request.param
+
+
+@pytest.fixture(scope="class")
+def created_model_registry_for_catalog(admin_client: DynamicClient, model_registry_namespace: str) -> ModelRegistry:
```

Comment:
is there any difference with `model_registry_instance` ? Model registry should be a normal model registry isn't it?

Reply:
teardown would cause problem if I use model_registry_instance. It expects the MR created to exist at the end. So I need a different one, to ensure teardown does not cause failure.